<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMain</class>
 <widget class="QMainWindow" name="StudentMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>19</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>                                      计算机与通信工程学院欢迎你</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>280</y>
             <width>91</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>图书馆&amp;云影湖</string>
           </property>
          </widget>
          <widget class="QLabel" name="Second">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>221</width>
             <height>281</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>280</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>长理爱的见证</string>
           </property>
          </widget>
          <widget class="QLabel" name="Thred">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>221</width>
             <height>281</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>280</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>长沙理工大学</string>
           </property>
          </widget>
          <widget class="QLabel" name="First">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>221</width>
             <height>281</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>申请信息</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>个人信息</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>退出</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_14"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>消息</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_17"/>
    <addaction name="separator"/>
    <addaction name="action_19"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>申请奖学金</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>申请记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>联系管理员</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>关于系统</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>个人信息</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>智育成绩</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>奖学金详情</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
